pkgs/by-name/gs/gsad/package.nix

```
@@ -0,0 +1,146 @@
+{
+  cmake,
+  doxygen,
+  fetchFromGitHub,
+  gettext,
+  git,
+  glib,
+  gnutar,
+  gnutls,
+  gvm-libs,
+  lib,
+  libidn2,
+  libgcrypt,
+  libmicrohttpd,
+  libselinux,
+  libsepol,
+  libtasn1,
+  libxml2,
+  libxslt,
+  p11-kit,
+  pcre2,
+  pkg-config,
+  python3Packages,
+  stdenv,
+  util-linux,
+  xmltoman,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "gsad";
+  version = "22.10.0";
+
+  # web app needed to start web server
+  src = fetchFromGitHub {
+    owner = "greenbone";
+    repo = "gsa";
+    rev = "refs/tags/v23.1.1";
+    hash = "sha256-yA+B6fh8C07tGEoy/+FPvcvx3D1OfPig0Eoqv2oJfsA=";
+  };
+
+  # web server
+  src2 = fetchFromGitHub {
+    owner = "greenbone";
+    repo = "gsad";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-/Ew1o/2Xve1VN8BlR8n+IaEfY8ecs9vBe5Cdx2ttZIo=";
+  };
+
+  unpackPhase = ''
+    runHook preUnpack
+
+    mkdir -p $out/share/gvm/gsad/web
+    mkdir -p $out/{gsad,build}
+    cp -R $src/* $out/share/gvm/gsad/web
+    cp -R $src2/* $out/gsad
+
+    runHook postUnpack
+  '';
+
+  preConfigure = ''
+    mkdir -p \
+      $out{bin,sbin,lib,include,share} \
+      $out/var/lib/gvm/{openvas/plugins,cert-data,users,scap-data,CA,data-objects,private/CA} \
+      $out/var/log \
+      $out/run/{gvmd,gsad} \
+      $out/share/gvm/gsad
+      #installManPage $src2/doc/gsad.8
```

```suggestion
      #installManPage $src2/doc/gsad.8
```
What is supposed to be done here?

---

I don't believe I need this line again but I will double check. If the result of the build produced the man output.

---

Man files is separate. For example, when building containers, you don't want to include man files in the container. Etc.